plugins/b2a-agents/skills/SKILL.md

```
@@ -0,0 +1,261 @@
+  for agent-to-agent data sales, evaluating if a data niche is suitable for agents,
+  or when asked to "build something agents would pay for".
+---
+
+# B2A Agents: Building for Agent Consumers
+
+Build x402 agents that other AI agents need and will pay for.
+
+## Why B2A Wins
+
+| Consumer (B2C) | Agent (B2A) |
+|----------------|-------------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)